jujugui/static/gui/src/app/components/header-search/_header-search.scss

```diff
@@ -68,8 +68,12 @@
     &__input {
         width: 136px;
         background-color: transparent;
-        border: none;
+        border: none !important;
```

Comment:
Yes this is a bug but also you caught me being lazy. I'll work out the important's.